### PR TITLE
Add high contrast logo

### PR DESCRIPTION
**Pull request by mchv**
[Enable to add high contrast logo](https://github.com/guardian/content-api-models/pull/64/files)

### CODE REVIEW

**Comment by shtukas** [comment]
:+1:

**Comment by shtukas** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 10.4
```
Comment (LATaylor-guardian) [inline CHANGELOG.md]: Why is this 10.4? Should it not be 10.3? The numbers look off from the version.

Comment (shtukas) [inline CHANGELOG.md]: Didn't it mean that updating the CapiModelsVersion to version 10.3 triggered the content-api-scala-client to move to 10.4 ?
(Or did I get something wrong ?)

Comment (mchv) [inline CHANGELOG.md]: The `client` and the `models` are not sync. For the model  we upgraded to `10.0` when we removed json4s and used only circe encoders and. For the `client` we upgraded to `10.0` when we removed the `useThrift` option. So `sponsor logos` which was models `9.25` has been added as a dependency in `client` `10.1`. The offset by `1` is unfortunate.

**Pull request merged**